CONTRIBUTING.md

```diff
@@ -259,14 +241,14 @@ pip install -r docs/requirements.txt
 
 Then, from the root of the repository, run:
 ```sh
-sphinx-build docs/source docs/build
+python docs/make_api_index.py && sphinx-build docs/source docs/build
 ```
 
 You can view the local build by opening `docs/build/index.html` in a browser.
 To refresh the documentation, after making changes, remove the `docs/build` folder and re-run the above command:
 
 ```sh
-rm -rf docs/build && sphinx-build docs/source docs/build
+rm -rf docs/build && python docs/make_api_index.py && sphinx-build docs/source docs/build
 ```
 
 To check that external links are correctly resolved, run:
```

Comment:
We could include the alternative `make` commands with fancy tabs.
```
::::{tab-set}

:::{tab-item} All platforms
python docs/make_api_index.py && sphinx-build docs/source docs/build
:::

:::{tab-item} Unix platforms with `make`
make html
::: 

::::
```

(Feel free to edit the headings, not sure if what I wrote is correct)

Reply:
I don't think that fancy tab formatting will work in the CONTRIBUTING guide, because that is just a normal markdown file included directly from the repo root. Maybe it will render properly on the website, but some people may directly open CONTRIBUTING.md on GitHub, and such syntax will appear jarring.

That said we should definitely mentioned the make commands in the contributing guide, even if they are not in tabs.

Reply:
Note the `make` commands only work if one is inside the `docs` folder.

Reply:
We introduced fancy tabs in the CONTRIBUTING guide in PR #231 . Should we remove this?

Reply:
Oh I hadn't realised this. I don't feel very strongly about it, what do you think?

This is how it renders on GitHub:
![Screenshot 2024-07-10 at 18 04 53](https://github.com/neuroinformatics-unit/movement/assets/20923448/f0902e8e-ee40-4260-8c15-a8bc99aad74d)

I often read contributing guides on github and I think others might do the same?

Reply:
maybe we can add a line at the top of the readme saying something like
"To see a nice rendering of this CONTRIBUTING guide, click [here](https://movement.neuroinformatics.dev/community/contributing.html#target-contributing)" .

Ideally we would omit that line from the rendering, but not sure if that is possible

Reply:
There is a way to include/exclude specific lines of the contributing guide. That's what I was doing long ago to deal with the difference in rendering.

Essentially it involves creating a `contributing_link.rst` in the docs source, whose content is something like the following (taken from a much earlier version of `movement`):

```rst
.. include:: ../../CONTRIBUTING.md
   :parser: myst_parser.sphinx_
   :end-before: **Contributors

.. important::
   .. include:: ../../CONTRIBUTING.md
      :parser: myst_parser.sphinx_
      :start-after: How to Contribute
      :end-before: ## Contributing code

.. include:: ../../CONTRIBUTING.md
   :parser: myst_parser.sphinx_
   :start-after: from the discussion.
```

But this gets quite finicky, as you have to make sure that the keywords (or line number) by which you define the included/omitted segments remain stable. Eventually I got tired of taking care of this, and decided that the contributing guide will be plain vanilla markdown. 

But I'm open to better suggestions.

Reply:
maybe we can also live with the fact that the rendering shows the line 
"To see a nice rendering of this CONTRIBUTING guide, click [here](https://movement.neuroinformatics.dev/community/contributing.html#target-contributing)" .

Reply:
Nah I think that's the most awkward option, as people that click on that while on the website will end up on the same page (and may even think the link is broken). I'd say we either:
- keep fancy rendering in the contributing guide and don't care about the few souls that wander into the guide's source on github
- get rid of all fancy rendering in the contributing guide.

I'm fine with either of the above options. All other choices seem a bit convoluted and hacky.

Reply:
Re option 1, our "Join the movement" section on both README and website clearly direct people towards the website version of the contributing guide.

Reply:
In the end we went wit option 1, i.e. keep the fancy rendering with tabs etc in `CONTRIBUTING.md`.

Reply:
> Re option 1, our "Join the movement" section on both README and website clearly direct people towards the website version of the contributing guide.

Yup, for this reason.